io.openems.edge.meter.virtual/src/io/openems/edge/meter/virtual/symmetric/add/SymmetricVirtualAdd.java

```diff
@@ -0,0 +1,126 @@
+package io.openems.edge.meter.virtual.symmetric.add;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+import org.osgi.service.event.propertytypes.EventTopics;
+import org.osgi.service.metatype.annotations.Designate;
+
+import io.openems.common.channel.AccessMode;
+import io.openems.common.exceptions.OpenemsError.OpenemsNamedException;
+import io.openems.edge.common.component.ComponentManager;
+import io.openems.edge.common.component.OpenemsComponent;
+import io.openems.edge.common.event.EdgeEventConstants;
+import io.openems.edge.common.modbusslave.ModbusSlave;
+import io.openems.edge.common.modbusslave.ModbusSlaveTable;
+import io.openems.edge.meter.api.MeterType;
+import io.openems.edge.meter.api.SymmetricMeter;
+import io.openems.edge.meter.api.VirtualMeter;
+import io.openems.edge.meter.virtual.common.AbstractVirtualAddMeter;
+
+@Designate(ocd = Config.class, factory = true)
+@Component(name = "Meter.Virtual.Symmetric.Add", //
+		immediate = true, //
+		configurationPolicy = ConfigurationPolicy.REQUIRE //
+) //
+@EventTopics({ //
+		EdgeEventConstants.TOPIC_CYCLE_BEFORE_PROCESS_IMAGE //
+})
+public class SymmetricVirtualAdd extends AbstractVirtualAddMeter<SymmetricMeter>
+		implements VirtualMeter, SymmetricMeter, OpenemsComponent, ModbusSlave {
+
+	private final SymmetricChannelManager channelManager = new SymmetricChannelManager(this);
+	private final List<SymmetricMeter> meters = new CopyOnWriteArrayList<>();
+
+	@Reference
+	protected ComponentManager componentManager;
+
+	private Config config;
+
+	@Reference
+	private ConfigurationAdmin configurationAdmin;
+
+	@Reference(//
+			policy = ReferencePolicy.DYNAMIC, //
+			policyOption = ReferencePolicyOption.GREEDY, //
+			cardinality = ReferenceCardinality.MULTIPLE, //
+			target = "(&(enabled=true)(!(service.factoryPid=Meter.Virtual.Symmetric.Add)))")
```

Comment:
Why is this limitation required? `Meter.Virtual.Symmetric.Add`. This way I could never _stack_ multiple virtual meters. I think this could be a usecase.